src/dialog/savedqueries/dlgsavedquerieseditor.cpp

```diff
@@ -0,0 +1,50 @@
+#include <QSqlTableModel>
+
+#include "dialog/savedqueries/dlgsavedquerieseditor.h"
+#include "dialog/savedqueries/savedqueriestablemodel.h"
+#include "library/libraryfeature.h"
+#include "library/trackcollection.h"
+
+DlgSavedQueriesEditor::DlgSavedQueriesEditor(LibraryFeature* pFeature,
+                                             TrackCollection* pTrackCollection,
+                                             QWidget* parent)
+        : QDialog(parent),
+          m_pTrackCollection(pTrackCollection),
+          m_pFeature(pFeature) {
+    setupUi(this);
+    SavedQueriesTableModel *pSaveModel = 
```

Comment:
@daschuer should we use unique_ptr here? Generally only classes inheriting from QT GUI controls like QWidget should be raw pointers, correct?